### PR TITLE
Updated slsn

### PR DESCRIPTION
**Pull request by erusseil**
Major update for the SLSN module.

The module now uses rainbow rather than independent fits. It currently requires the installation of this fork of the light-curve package: https://github.com/erusseil/light-curve-python/tree/extended

The AGN module, not used anywhere and with no plans for the future, has been removed. The SLSN partly relied on functions from the AGN code. They are now all included inside the SLSN module, making it much more readable.

A new temporary (bad) joblib model is provided, just to make sure the pipeline is working. The true model will be upload later, after the training is complete.

### CODE REVIEW

File: fink_science/slsn/basic_functions.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    """
+
+    if len(x) == 0:
+        return -1
+
+    elif absolute:
+        return max(x, key=abs)
+
+    else:
+        return x.max()
```
Comment (JulienPeloton) [inline fink_science/slsn/basic_functions.py:127]: Better to use `np.max(x)` as x might be a `list`

**Review by JulienPeloton — COMMENTED** [review]
Thanks @erusseil for the PR. I could test the module on about 1000 alerts from elasticc. 

The rate measured is about 10 alerts/second, completely dominated by `fitter._eval` in the feature extraction. I benchmarked rainbow, and this makes sense (expectation is 0.1s / alert). So this means you cannot speed up further on your side, except by filtering out incoming alerts (for the future!)

However, the installation of the modified version of `light-curve-python` was a pain. I cannot afford this on production in the cloud (it requires so many things to recompile `light-curve`). Do you think you could merge your development into the main repo, so that Kostya can make a proper release with your changes?

**Comment by erusseil** [comment]
Thank you for the review @JulienPeloton. We are working on the merge to the main light-curve repo, hopefully it should happen very soon. We are having some obscure issues with iminuit right now :/

**Comment by JulienPeloton** [comment]
Thanks!

**Comment by erusseil** [comment]
Done, the branch has been merged to the master light-curve package.

**Comment by erusseil** [comment]
The model is now updated to produce meaningful results from the main branch of the light-curve package. If everything runs on your side, it is ready to recompute results.

**Comment by JulienPeloton** [comment]
Thanks @erusseil -- I'm merging into another branch to continue the integration with other models.

**Pull request merged**